tests/src/utils.rs

```diff
@@ -172,10 +172,18 @@ impl TestProcess {
         assert!(self.get_stdout().await.contains(string));
     }
 
+    pub async fn assert_stdout_doesnt_contain(&self, string: &str) {
+        assert!(!self.get_stdout().await.contains(string));
+    }
+
     pub async fn assert_stderr_contains(&self, string: &str) {
         assert!(self.get_stderr().await.contains(string));
     }
 
+    pub async fn assert_stderr_doesnt_contain(&self, string: &str) {
```

Comment:
This is not being used?

Reply:
no, but i figured if i was adding the other one it might be useful at some point